tests/model_serving/model_server/kserve/negative/test_rapid_isvc_create_delete.py

```
@@ -0,0 +1,258 @@
+            wait_timeout=Timeout.TIMEOUT_2MIN,
+            sleep=3,
+            func=_isvc_gone,
+        ):
+            if gone:
+                LOGGER.info(
+                    "Rapid lifecycle: ISVC confirmed deleted",
+                    cycle=cycle_index,
+                    name=isvc_name,
+                )
+                return
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)